SwiftScripts/Sources/Codegen/main.swift

```diff
@@ -8,27 +8,13 @@ import TargetConfig
 // target, so we're using an arg parser to figure out which one to build,
 // and an enum to hold related options.
 struct Codegen: ParsableCommand {
-
-  enum ArgumentError: Error, LocalizedError {
-    case invalidTargetName(name: String)
-    case invalidPackageType(name: String)
-
-    var errorDescription: String? {
-      switch self {
-      case let .invalidTargetName(name):
-        return "The target \"\(name)\" is invalid. Please try again."
-
-      case let .invalidPackageType(name):
-        return "The package type \"\(name)\" is invalid. Please try again."
-      }
-    }
-  }
-
   @Option(
+    wrappedValue: [],
     name: [.customLong("target"), .customShort("t")],
-    help: "The target to generate code for - Required."
+    parsing: .upToNextOption,
+    help: "The target to generate code for."
   )
-  var targetName: String
+  var targetNames: [String]
```

Comment:
This allows us to pass multiple targets to run codegen on at once. And if you pass nothing, we will generate all targets.